src/index.js

```diff
@@ -7,18 +9,76 @@ const orders = (time, product, table) => {
   });
 }
 
+const urlApi= 'https://us-central1-escuelajs-api.cloudfunctions.net/orders';
+
 const menu = {
   hamburger: 'Combo Hamburguesa',
   hotdog: 'Combo Hot Dogs',
   pizza: 'Combo Pizza',
 };
 
-const table = ['Mesa 1', 'Mesa 2', 'Mesa 3', 'Mesa 4', 'Mesa 5'];
+const tableList = ['Mesa 1', 'Mesa 2', 'Mesa 3', 'Mesa 4', 'Mesa 5'];
 
 const waiter = () => {
-  orders(6000, menu.hamburger, table[3])
+  orders(geTimeRandom(), menu.hamburger, tableList[3])
     .then((res) => console.log(res))
     .catch((err) => console.error(err));
 };
 
 waiter();
+
+const waiter2 = () => {
+  orders(geTimeRandom(), menu.hotdog, tableList[0])
+    .then((res) => console.log(res))
+    .catch((err) => console.error(err));
+  orders(geTimeRandom(), menu.pizza, tableList[2])
+    .then((res) => console.log(res))
+    .catch((err) => console.error(err));
+};
+
+waiter2();
+
+
+ const waiter3 = async () => {
+   try {
+     const orders3 = await orders(geTimeRandom(), menu.hotdog, tableList[1])
+     console.log(orders3);
+   }
+   catch(err){
+     console.log(err)}
+};
+
+waiter3();
+
+
+
+async  function fetchOrders ()  {
+  let response = await fetch(urlApi);
+  let dataOrder = await response.json();
+  return  dataOrder.data;
+};
+
+const waiter4 = async () => {
+  try {
+    for(let x=0;x<=3;x++){
```

Comment:
No entendi para que es esta implementación...

Reply:
Para las 4 peticiones de menus a los meseros, utilicé un for en vez de hacerlas una por una.